packages/blockchain-wallet-v4-frontend/src/data/modules/addressesBch/sagas.js

```diff
@@ -12,10 +12,22 @@ export default ({ coreSagas, networks }) => {
   const editBchAccountLabel = function * (action) {
     try {
       const { index, label } = action.payload
+      const wallets = (yield select(
+        selectors.core.common.bch.getAccountsBalances
```

Comment:
I think the more appropriate selector would be `selectors.core.common.bch.getActiveHDAccounts`